modules/ROOT/nav.adoc

```diff
@@ -334,6 +334,10 @@
 **** xref:reference:rpk/rpk-registry/rpk-registry-compatibility-level.adoc[]
 ***** xref:reference:rpk/rpk-registry/rpk-registry-compatibility-level-get.adoc[]
 ***** xref:reference:rpk/rpk-registry/rpk-registry-compatibility-level-set.adoc[]
+**** xref:reference:rpk/rpk-registry/rpk-registry-mode.adoc[]
+***** xref:reference:rpk/rpk-registry/rpk-registry-mode-get.adoc[]
+***** xref:reference:rpk/rpk-registry/rpk-registry-mode-reset.adoc[]
+***** xref:reference:rpk/rpk-registry/rpk-registry-mode-set.adoc[]
```

Comment:
Could we change the order so that set is above reset?

Reply:
Not really as it's alphabetical order.